.github/workflows/ci.yml

```diff
@@ -45,10 +45,15 @@ jobs:
         run: sbt cli/docker
 
       - run: |
-          git clone https://github.com/circe/circe.git
+          set -eu
+          for REPO in circe/circe indeedeng/proctor
```

Comment:
Is it worth making this a matrix build so these can fail independently? Also in parallel

Reply:
I can make this parallel within the job itself, but given that I hope to have 5-10 repos in here, I think this will make matrix build too big - and each job will have to suffer the penalty of publishing docker CLI.

In a next PR I will make this faster by using GNU parallel and mounting cache folders into the containers.

Reply:
Sounds good. It's very valuable to have a complete e2e testing pipeline but if the job becomes too slow then we can consider making it optional in PRs behind some GH label. Many PRs don't influence auto-indexing stuff and it sucks when the CI takes +30 minutes